src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let rentalCost = 0;
+
+  if (days < 3) {
+    rentalCost = days * 40;
+
+    return rentalCost;
+  }
+
+  if (days >= 3 && days < 7) {
+    rentalCost = (days * 40) - 20;
+
+    return rentalCost;
+  }
+
+  if (days >= 7) {
+    rentalCost = (days * 40) - 50;
+
+    return rentalCost;
```

Comment:
If you return "rentalCost" in every statement you don't need this variable, you can just return the result of the calculations.